packages/backend/src/clients/EmailClient/templates/invitation.html

```diff
@@ -871,9 +871,9 @@
                                                                                                         mso-line-height-rule: exactly;
                                                                                                         mso-text-raise: 8px;
                                                                                                     "
-                                                                                                        href="{{inviteUrl}}"
-                                                                                                        target="_blank"
-                                                                                                >Join
+                                                                                                    href="{{inviteUrl}}"
```

Comment:
unrelated change ?

Reply:
just ran `yarn workspace backend fix-format` and this happened